src/main/java/Event.java

```diff
@@ -0,0 +1,42 @@
+import java.time.LocalDateTime;
+
+public class Event extends Task {
+    /** Start datetime of the Event. */
+    protected LocalDateTime start;
```

Comment:
Since there are no classes inheriting from Event, should we set it to `private` first?

Reply:
To add on, perhaps can consider abstracting Event and Deadline together as a abstract TimedTask.